src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/public/index.html

```diff
@@ -36,61 +36,6 @@
 </head>
 
 <body>
-  <script>
-    (function ($) {
-      $.fn.datepicker.dates['no'] = {
-        days: ['søndag', 'mandag', 'tirsdag', 'onsdag', 'torsdag', 'fredag', 'lørdag'],
-        daysShort: ['søn', 'man', 'tir', 'ons', 'tor', 'fre', 'lør'],
-        daysMin: ['sø', 'ma', 'ti', 'on', 'to', 'fr', 'lø'],
-        months: ['januar', 'februar', 'mars', 'april', 'mai', 'juni', 'juli', 'august', 'september', 'oktober',
-          'november', 'desember'
-        ],
-        monthsShort: ['jan', 'feb', 'mar', 'apr', 'mai', 'jun', 'jul', 'aug', 'sep', 'okt', 'nov', 'des'],
-        today: 'i dag',
-        monthsTitle: 'Måneder',
-        clear: 'Nullstill',
-        weekStart: 1,
-        format: 'dd.mm.yyyy'
-      };
-    }(jQuery));
-
-    (function ($) {
-      $.fn.datepicker.dates['nn'] = {
-        days: ['sundag', 'måndag', 'tysdag', 'onsdag', 'torsdag', 'fredag', 'laurdag'],
-        daysShort: ['sun', 'mån', 'tys', 'ons', 'tor', 'fre', 'lau'],
-        daysMin: ['su', 'må', 'ty', 'on', 'to', 'fr', 'la'],
-        months: ['januar', 'februar', 'mars', 'april', 'mai', 'juni', 'juli', 'august', 'september', 'oktober',
-          'november', 'desember'
-        ],
-        monthsShort: ['jan', 'feb', 'mar', 'apr', 'mai', 'jun', 'jul', 'aug', 'sep', 'okt', 'nov', 'des'],
-        today: 'i dag',
-        monthsTitle: 'Månadar',
-        clear: 'Nullstill',
-        weekStart: 1,
-        format: 'dd.mm.yyyy'
-      };
-    }(jQuery));
```

Comment:
Do we need to do this for all the languages we support, or is it just norwegian thats missing?

Reply:
The previous datepicker needed this, but this has now been deleted  ;) 
So now the moment-util handles all localisation. Except for that one exception which I have overrided.